File: exercice2/main.ts
class Furniture {
    editor: string;
    designerName: string;
    year: number;
   
   constructor(editor: string, designerName: string, year: number) {
       this.editor = editor;
       this.designerName = designerName;
       this.year = year;
   }
   
   }
   
   let WassilyChair: Furniture = new Furniture("Knoll", "Marcel Breuer", 1925);
   
   
   console.log(WassilyChair);
   
   WassilyChair.year = 1926;
   
   console.log(WassilyChair.year)

File: exercice2/main.java
class Furniture {
    String editor;
    String designerName;
    int year;
    
    Furniture(String editor, String designerName, int year) {
        this.editor = editor;
        this.designerName = designerName;
        this.year = year;
    }
}


class Playground {
    public static void main(String[ ] args) {
    Furniture wassilyChair = new Furniture("Knoll", "Marcel Breuer", 1925);
        System.out.println("The Wassily Chair was created by " + wassilyChair.designerName + " and edited by " + wassilyChair.editor + " in " + wassilyChair.year + ".");
        wassilyChair.year = 1926;
        System.out.println(wassilyChair.year);

    }
}
